Tests/KlaviyoSwiftTests/StateManagementTests.swift

```diff
@@ -355,45 +355,71 @@ class StateManagementTests: XCTestCase {
         initialState.flushing = false
         let store = TestStore(initialState: initialState, reducer: KlaviyoReducer())
 
-        let profileActions: [(Profile.ProfileKey, Any)] = [
-            (.city, "Sharon"),
-            (.region, "New England"),
-            (.address1, "123 Main Street"),
-            (.address2, "Apt 6"),
-            (.zip, "02067"),
-            (.country, "Mexico"),
-            (.latitude, 23.0),
-            (.longitude, 46.0),
-            (.title, "King"),
-            (.organization, "Klaviyo"),
-            (.firstName, "Jeffrey"),
-            (.lastName, "Lebowski"),
-            (.image, "foto.png"),
+        let profileAttributes: [(Profile.ProfileKey, Any)] = [
```

Comment:
Updated this existing test since it was already doing what we wanted except for two things - 
1. Making sure that `pendingProfile` was cleared after adding it to the token request 
2. Making sure that the token request payload contains the profile properties set using the setProfileAttributes method.

Added both of these into this test.

Reply:
Yeah this works for me, thanks!